backend/private-graph/graph/schema.resolvers.go

```diff
@@ -4327,14 +4328,19 @@ func (r *mutationResolver) DeleteSessions(ctx context.Context, projectID int, pa
 		return false, err
 	}
 
-	_, err = r.StepFunctions.DeleteSessionsByQuery(ctx, utils.QuerySessionsInput{
-		ProjectId:    projectID,
-		Email:        email,
-		FirstName:    firstName,
-		Params:       params,
-		SessionCount: sessionCount,
-		DryRun:       util.IsDevOrTestEnv(),
-	})
+	if util.IsInDocker() {
+		deleteHandlers := delete_handlers.InitHandlers(r.DB, r.ClickhouseClient, nil, r.StorageClient)
+		deleteHandlers.DeleteSessions(ctx, projectID, params.DateRange.StartDate, params.DateRange.EndDate, params.Query)
+	} else {
```

Comment:
would be nice to create an e2e test case for this, validating the session no longer shows up once deleted. relatively big effort to add a functional test like that so perhaps we just ticket that for the future?

Reply:
:+1: ticketed